com.woltlab.wcf/option.xml

```diff
@@ -1719,5 +1714,6 @@ DESC:wcf.global.sortOrder.descending</selectoptions>
 		<option name="register_password_must_contain_upper_case" />
 		<option name="register_password_must_contain_digit" />
 		<option name="register_password_must_contain_special_char" />
```

Comment:
Remove the old entries. Should probably happen in `master`, because several open PRs already adjust this part of option.xml.